### PR TITLE
Fix spell enum in MagicAction

### PR DESCRIPTION
**Pull request by richiebres**

### CODE REVIEW

File: src/game/actions.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export interface MoveAction {
 
 export interface MagicAction {
   type: "MAGIC";
-  spell: Spell;
+  spell: "Spell";
```
Comment (pwerry) [inline src/game/actions.ts:15]: Shouldn't this be a type so we can load various types of Spells?

File: src/game/actions.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export interface MoveAction {
 
 export interface MagicAction {
```
Comment (pwerry) [inline src/game/actions.ts:13]: Maybe we need to add `requiredMagicLevel` to `Spell` here

File: src/game/actions.ts (hunk 1 above)
Comment (jeffrey-ng) [reply src/game/actions.ts:15]: This was actually fixed here: https://github.com/TheUnblockedDemo/2048/commit/9e73f2f3a4a9fed154e17c6eb4cd462f46406ebd

Was missing an import for this enum that already exists.

File: src/game/actions.ts (hunk 2 above)
Comment (richiebres) [reply src/game/actions.ts:13]: The `magicLevel` is already in the base class.

One of TypeScript’s core principles is that type checking focuses on the shape that values have. This is sometimes called “duck typing” or “structural subtyping”. In TypeScript, interfaces fill the role of naming these types, and are a powerful way of defining contracts within your code as well as contracts with code outside of your project.

**Pull request merged**